src/main/java/gui/Controller.java

```diff
@@ -183,4 +212,82 @@ final void proceedToMonitorScene(final MonitorScene ms, final Stage primaryStage
                 primaryStage, stylesheet));
     }
 
+    /**
+     * Changes the time of the timer.
+     *
+     * @param elapsedTime the elapsed time
+     */
+    public void changeTime(final long elapsedTime) {
+        timerLabel.setText(getTimeString(elapsedTime));
+    }
+
+    /**
+     * Add information to correct VBox.
+     *
+     * @param text The text to add.
```

Comment:
We should not use capital letters for the parameter description I think, as we never do this.